src/api.py

```diff
@@ -0,0 +1,123 @@
+from flask import Flask, request, jsonify
+from flasgger import Swagger
+from flask_cors import CORS
+import json
+from . import processing as p
+
+app = Flask(__name__)
+
+# allow cross-origin sharing to use http
+CORS(app)
+
+# adding Swagger
+app.config['SWAGGER'] = {
+    'title': 'Orders API',
+    'uiversion': 3
+}
+Swagger(app, template_file='swagger.yaml')
+
+@app.route('/api/load_table', methods=['GET'])
+def load_table():
+    """Load data from database"""
+    table_name = request.args.get('table_name')
+    engine = p.create_engine()
+    result = p.load_data_from_db(table_name, engine)
+    return jsonify(json.loads(result.write_json()))
```

Comment:
Can you standardize your response? Something like:

```
jsonify({"status": "success", "message": _whatever_you_want_to_send})
```

I also do not see error handling here, seems you're very optimistic with the code. Haha, the other status can be error for failure or other scenarios

Lastly add status codes to your responses

Reply:
Done

Reply:
Done